Update QB2 specs/setup content and remove preproduction notes (#158)
tenstorrent/tenstorrent.github.io: core/systems/quietbox/quietbox-bh-2/specifications.md

| Shipping Box Dimensions | Height: 20.7” (52.5 cm) Width: 14.4” (36.7 cm) Depth: 25.0” (63.5 cm) |
| Shipping Box Weight | 23.2 kg (52 lbs)  |

## **Blackhole p300 Card Specifications**

> TT-QuietBox 2 is powered by two p300c cards for a total of four Blackhole chips. The table below describes the specifications of a single p300c card for your reference. Please note the p300c card is not sold separately outside of the QuietBox 2. If you are interested in a Blackhole card to add to your existing system, check out the available cards at www.tenstorrent.com/cards

Reviewer: ```suggestion
TT-QuietBox 2 is powered by two p300c cards for a total of four Blackhole chips. The table below describes the specifications of a single p300c card for your reference. Please note the p300c card is not sold separately outside of the QuietBox 2. If you are interested in a Blackhole card to add to your existing system, check out our [available cards](https://www.tenstorrent.com/cards).
```
